scenario_09_sandbox.lua

```diff
@@ -38376,151 +38382,236 @@ end
 -- -MAIN FROM KOSAI	F	initialGMFunctions
 -- -CUSTOM				F	customButtons
 -- -ONE-OFFS			F	oneOffs
+snakeCallSigns = {
+	"Żmija", "Wąż", "Gniewosz", "Zaskroniec", "Grzechotnik", "Dusiciel", "Kobra"
```

Comment:
Did you verify that those special characters (like ą) are supported in EE?

Reply:
You yourself saw them on your screen yesterday I would hope